### ISSUE TITLE
MODULE - Radio Repair 

### ISSUE THREAD

**Issue by maliarich** [maliarich/EMTTARU-Doc#5]
###  **Radio Repair** 

A radio receiver is an electronic device that receives audio signals through an antenna, then amplifies the resulting sound and delivers it to your ears through speakers. The signals are either amplitude modulated (AM) or frequency modulated (FM)
![image](https://github.com/maliarich/EMTTARU-Doc/assets/56769901/ca1b356c-9d42-4957-ac07-4df4cc8f8197)

**Comment by maliarich** [comment]
**How Can I Identify a Radio Problem?**

- If the radio won’t work, make sure the power is on at the outlet and test the electrical cord and replace if needed.
- If the station indicator won’t move, check the pointer (see below). Less expensive radios use a station frequency pointer mounted on a dial cord that’s moved by a small wheel on the side of the radio. Better radios use a digital frequency readout that cannot be adjusted.
![image](https://github.com/maliarich/EMTTARU-Doc/assets/56769901/7c1f9005-ce4f-4cdc-82e5-b2cd262d0554)

- If the sound if fuzzy, disassemble the unit and locate the volume control spray electrical contact cleaner into the control and rotate or slide the control several times to lubricate the mechanism.

_**Don’t throw away the owner’s manual that came with your radio. It not only tells you how to maintain it, it also tells you where to get parts. In addition, it probably includes warranty information to determine whether you need to fix it yourself or return it.**_

**What Do I Need for Radio Repair?**

Most modern radios are built on printed circuit boards that can’t be repaired by anyone but trained technicians. However, there still are tests you can perform and parts you can replace. Check your local Radio Shack or other electronics supplier for parts and tools. Here are a few tools you may need:

- Screwdrivers
- Wrenches
- Electronic contact cleaner
- Denatured alcohol
- Soldering iron and solder 
https://openculture.agency/outcomes/askotec/

_What Are the Steps to Radio Repair?_

**Reset a radio dial pointer:**

- Turn on the radio and find a strong radio signal somewhere near the middle of the dial. Listen for the frequency (such as 740 AM or 96.5 FM).
- Use a piece of tape or grease pencil to mark the dial for where the pointer should be for that frequency.
- Unplug the receiver and open the housing. Typically, there are two to four screws accessed from the underside of the radio.
- Inspect the pointer mechanism for obvious problems, such as a cord that came off the end rollers or a break in the cord. If the cord is broken, replace or repair it.
- Loosen the spring clips, then move the pointer along the dial cord until the station being received lines up with the tape or mark on the dial.
- Tighten the clips on the cord, making sure you don’t move the pointer.
Plug the radio in and verify that the dial pointer reads correctly.

**Replace a radio external antenna**

- Loosen the radio antenna screw on the radio’s exterior and remove the antenna. Replace it with an exact replacement unit available through the manufacturer or an electronics store.
- If there is no exterior screw, you will need to open the radio to access the antenna base
- If the unit doesn’t work, check and replace batteries, clean corroded battery contacts with fine sandpaper, and carefully adjust battery contacts as needed to improve battery performance. Also spray electrical contact cleaner into the jack and wipe excess with a cotton swab. If that doesn’t solve the problem, open the unit and inspect the jack for broken connections; re-solder connections as needed.
- If the sound is intermittent, look for corroded or bent contacts in the headphone plug or jack. Also check for a broken headphone wire and repair or replace the wire if you find one broken. Check for a loose or broken external antenna.
- If the sound is fuzzy, try the headphones in another unit and replace the headphones if necessary.
- If the function buttons don’t work, clean any dirty button mechanisms with electrical contact cleaner, then carefully lubricate them with white lithium spray.

**Replace a radio internal antenna:**

- Unplug the receiver and open the housing. Typically, there are two to four screws accessed from the underside of the radio.
- Find and remove the internal antenna (or the internal base of the external antenna). Internal antennas typically are numerous wires wound around a flat piece, then sealed to minimize moisture and damage.
- Inspect the wires wound around the antenna for obvious damage. Also make sure the antenna is firmly plugged into the main circuit board. If the antenna is damaged, replace it. If no replacement is available, attempt to repair the antenna, which is not easy because of all the fine wiring, or replace the radio as a unit.

_Clean radio electronic components_:

- Unplug the receiver and open the housing.
- Use canned air to blow dust from components including any power cords and speaker jacks.
- Use electrical contact cleaner or a cotton swab dipped in denatured alcohol to clean electronic components as needed.
- If you find broken components on a circuit board, replace the circuit board. As a last resort before purchasing a new circuit board, you can use a soldering iron and electronic solder to attempt a repair.

**Comment by maliarich** [comment]
### How Can I Identify a Radio Problem?

- If the radio won’t work, make sure the power is on at the outlet and test the electrical cord and replace if needed.
- If the station indicator won’t move, check the pointer (see below). Less expensive radios use a station frequency pointer mounted on a dial cord that’s moved by a small wheel on the side of the radio. Better radios use a digital frequency readout that cannot be adjusted.

**Comment by timmwille** [comment]
are these images yours?

**Comment by maliarich** [comment]
> are these images yours?

No... downloaded from online shop want to  replace it during the repair day event @timmwille
